src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -915,8 +915,10 @@ settings.set( "edit.default_extension", "lua" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
+settings.set( "filetype.txt", "edit {}" )
```

Comment:
Why `{}`? I'd personally go for `%s`, which means you can just do `settings.get(xyz):format(sCommand)`.